### PR TITLE
Matt caption

### LINKED ISSUE #2: Activity 1: Your First Caption
Using the API, I have just created a file with a random octocat for you to caption. In this activity, we will learn how to make changes to files in our GitHub repository.

- [ ] @beardofedu
- [ ] @GeorgeEatsData2018
- [ ] @orrelljim
- [ ] @dandrews999
- [ ] @ajaysjain
- [ ] @jaredweiss8
- [ ] @mattrt
- [ ] @sbvnaveen
- [ ] @mcranford
- [ ] @tjayadeva
- [ ] @rentingxu
- [ ] @sharduldatar
- [ ] @bprince-dapm
- [ ] @tjayadeva
- [ ] @tjayadeva
- [ ] @mattrt
- [ ] @graulm

### PR DESCRIPTION
**Pull request by mattrt**
Edit the caption to match the image (for GOLD).

Closes #2

### CODE REVIEW

File: _posts/0000-01-07-mattrt.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
```
Comment (beardofedu) [inline _posts/0000-01-07-mattrt.md]: Hey @mattrt maybe the `title` should be changed?

File: _posts/0000-01-07-mattrt.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
...
 ---
 
-Caption here
+G-OMG!!
```
Comment (beardofedu) [inline _posts/0000-01-07-mattrt.md]: nice reference to gold + excitement 👍

File: _posts/0000-01-07-mattrt.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ layout: slide
...
 
 ![minertocat](https://octodex.github.com/images/minertocat.png)
```
Comment (beardofedu) [inline _posts/0000-01-07-mattrt.md]:
```suggestion
![gangnam](https://octodex.github.com/images/gangnamtocat.png)
```

File: _posts/0000-01-07-mattrt.md (hunk 1 above)
Comment (mattrt) [reply _posts/0000-01-07-mattrt.md]: interesting, thanks...

@ajaysjain any thoughts here?

**Pull request merged**